src/main/java/github/weichware10/util/data/TrialData.java

```diff
@@ -22,44 +34,67 @@
+     * @param configId  - the configuration of the stored data
+     * @param startTime - Startzeitpunkt des Versuchs
      */
-    public TrialData(Enums.ToolType toolType, String trialId, String configId) {
+    @JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
+    public TrialData(@JsonProperty("toolType") Enums.ToolType toolType,
+            @JsonProperty("trialId") String trialId, @JsonProperty("configId") String configId,
+            @JsonProperty("startTime") DateTime startTime, @JsonProperty("answer") String answer,
+            @JsonProperty("data") List<DataPoint> dataPoints) {
         this.toolType = toolType;
         this.trialId = trialId;
         this.configId = configId;
-        this.startTime = DateTime.now();
-        this.dataPoints = new ArrayList<DataPoint>();
+        this.startTime = startTime;
+        this.answer = answer;
+        this.dataPoints = dataPoints;
     }
 
     /**
      * Stores the TrialData for the different tools internally.
      *
-     * @param toolType   - the tooltype of the stored data
-     * @param trialId    - the id of the trial
-     * @param configId   - the configuration of the stored data
-     * @param startTime  - Startzeitpunkt des Versuchs
-     * @param answer     - Anwort des Versuchs
-     * @param dataPoints - Daten des Versuchs
+     * @param toolType - the tooltype of the stored data
+     * @param trialId  - the id of the trial
+     * @param configId - the configuration of the stored data
      *
-     * @since v0.3
+     * @since v0.2
      */
-    public TrialData(Enums.ToolType toolType, String trialId, String configId,
-            DateTime startTime, String answer, List<DataPoint> dataPoints) {
+    public TrialData(Enums.ToolType toolType, String trialId, String configId) {
         this.toolType = toolType;
         this.trialId = trialId;
         this.configId = configId;
-        this.startTime = startTime;
-        this.answer = answer;
-        this.dataPoints = dataPoints;
+        this.startTime = DateTime.now();
+        this.dataPoints = new ArrayList<DataPoint>();
     }
 
+    // /**
+    //  * Stores the TrialData for the different tools internally.
+    //  *
+    //  * @param toolType   - the tooltype of the stored data
+    //  * @param trialId    - the id of the trial
+    //  * @param configId   - the configuration of the stored data
+    //  * @param startTime  - Startzeitpunkt des Versuchs
+    //  * @param answer     - Anwort des Versuchs
+    //  * @param dataPoints - Daten des Versuchs
+    //  *
+    //  * @since v0.3
+    //  */
+    // public TrialData(Enums.ToolType toolType, String trialId, String configId,
+    //         DateTime startTime, String answer, List<DataPoint> dataPoints) {
+    //     this.toolType = toolType;
+    //     this.trialId = trialId;
+    //     this.configId = configId;
+    //     this.startTime = startTime;
+    //     this.answer = answer;
+    //     this.dataPoints = dataPoints;
+    // }
+
```

Comment:
Warum ist das alles in Kommentaren ?

Reply:
Das habe ich erstmal auskommentiert, da sonst der Konstruktor doppelt gewesen wäre

Reply:
Kann ich aber auch löschen, da wir es nicht mehr benötigen